packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/aggregate_source.dart

```diff
@@ -0,0 +1,3 @@
+enum AggregateSource {
```

Comment:
Let's make sure these have code comments + license headers.

Reply:
Inline documentation is on the TODO